### ISSUE TITLE
Kicking User & Bandwidth Usage report

### ISSUE THREAD

**Issue by sah-anshu** [zhaojh329/wifidog-ng#24]
Hi

First Issue, The KICK Url is not working. The User is redirected to Login Page but still can access the Internet.
wget "http://lanip:2060/wifidog/ctl?op=kick&mac=0C:1D:AF:C4:DB:FC" -O /dev/null

Second Issue. 
Can we report the bandwidth Used by Any API ?

Thanks in Advance
Anshu

**Comment by zhaojh329** [comment]
Example:
wget 'http://192.168.1.1:2060/wifidog/ctl?op=kick&mac=0C:1D:AF:C4:DB:FC' -O /dev/null

**Comment by sah-anshu** [comment]
NO, this Url redirects to the Welcome page. and User doesn't get logged out.

**Comment by zhaojh329** [comment]
Please use the latest version.
